README.md

```diff
@@ -166,8 +166,21 @@ plan_hierarchy:
     data_hash: yaml_data
 ```
 
+
 See the following [documentation](https://puppet.com/docs/bolt/latest/hiera.html#outside-apply-blocks) for further explanation.
 
+#### Using indictor_exclusion with a code manager and hiera.yaml
+
+To utilize the indicator_exclusions with a code manager, place the plan_hierarchy listed in the step above, placing it in the root level [hiera.yaml](https://puppet.com/docs/puppet/latest/hiera_config_yaml_5.html) file.
```

Comment:
Phrasing change needed
You are not using it with code manager you are using hiera within the plan,  its the environment layer of hiera not root, and you should link to https://www.puppet.com/docs/pe/2023.4/writing_plans_in_puppet_language_pe.html#using_hiera_with_plans